chapter01/1.3 - URLify/urlify-4.js

```diff
@@ -0,0 +1,6 @@
+function urlify(s) {
+    s = s.split(' ').filter(el => el !== '').join('%20');;
```

Comment:
In most interviews, they'll want you to write the actual logic other than using in-built function like this, esp if that is part of the question and not just a utility.